aspnet/fundamentals/diagnostics.rst

```diff
@@ -3,85 +3,25 @@ Diagnostics
+	:local:
+	:depth: 1
 	
-Setting the ``ASPNET_ENV`` variable to anything other than Development (e.g. Production) will cause the application not to call ``UseDeveloperExceptionPage()`` and only a HTTP 500 response code with no message details will be sent back to the browser, unless an exception handler is configured such as ``UseExceptionHandler()``.
-
-We will cover the features provided by the error page in the next section (ensure ``ASPNET_ENV`` is reset to Development if you are following along).
-
-Using the error page during development
----------------------------------------
-
-The default error page will display some useful diagnostics information when an unhandled exception occurs within the web processing pipeline. The error page includes several tabs with information about the exception that was triggered and the request that was made. The first tab shows the stack trace:
-
-.. image:: diagnostics/_static/errorpage-stack.png
-
-The next tab shows the contents of the Querystring collection, if any:
-
-.. image:: diagnostics/_static/errorpage-query.png
-
-In this case, you can see the value of the ``throw`` parameter that was passed to this request. This request didn't have any cookies, but if it did, they would appear on the Cookies tab. You can see the headers that were passed, here:
-
-.. image:: diagnostics/_static/errorpage-headers.png
-
-.. note:: In the current pre-release build, the Cookies section of ErrorPage is not yet enabled. `View ErrorPage Source <https://github.com/aspnet/Diagnostics/blob/dev/src/Microsoft.AspNet.Diagnostics/Views/ErrorPage.cshtml>`_.
-
-HTTP 500 errors on Azure
--------------------------
-
-If your app throws an exception before the ``Configure`` method in *Startup.cs* completes, the error page won't be configured. The app deployed to Azure (or another production server) will return an HTTP 500 error with no message details. ASP.NET 5 uses a new configuration model that is not based on *web.config*, and when you create a new web app with Visual Studio 2015, the project no longer contains a *web.config* file. (See `Understanding ASP.NET 5 Web Apps <http://docs.asp.net/en/latest/conceptual-overview/understanding-aspnet5-apps.html>`_.)
-
-The publish wizard in Visual Studio 2015 creates a *web.config* file if you don't have one. If you have a *web.config* file in the *wwwroot* folder, deploy inserts the markup into the the *web.config* file it generates. 
-
-To get detailed error messages on Azure, add the following *web.config* file to the *wwwroot* folder.
-
-.. note:: Security warning: Enabling detailed error message can leak critical information from your app. You should never enable detailed error messages on a production app.
-
-.. code-block:: html
-
-	<configuration>
-	   <system.web>
-		  <customErrors mode="Off"/>
-	   </system.web>
-	</configuration>
+`Browse or download samples on GitHub <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/diagnostics/sample>`_.
```

Comment:
This should probably be consistent within the docs. Is this something we should make a policy, one way or the other? For me as a consumer of docs, I like to be able jump right to the code at the top if I can (or at least, know that I can if I want to as I start reading the article), but I may not be typical.